src/index.d.ts

```diff
@@ -1,5 +1,6 @@
 declare namespace uvu {
-	type Callback<T> = (context: T) => Promise<void> | void;
+	type Crumbs = { __suite__: string; __test__: string };
```

Comment:
tbh i would have envisioned this like:

```suggestion
	type Crumbs = { __suite__: {name: string}; __test__: {name: string} };
```

So we can push more things into that in the future—don't quite know what, but this is simple.

Reply:
We can cross that road once we get there, but I highly doubt there will ever be additional data to shove in. 
Plus, the entire `context` object _is_ for the suite/test. it's totally shared. The only thing that _differs_ over time is _which_ test is currently being executed – and that's managed by name since that's the only identifiable information.